classes/output/report_table.php

```diff
@@ -109,6 +109,7 @@ public function __construct(string $uniqueid, string $baseurl, int $page = 0,
         $this->currpage = $page;
         $this->pagesize = $perpage;
         $this->sortable(true);
+        $this->sort_default_column = 'timecreated DESC';
```

Comment:
Hi Peter, how about merging these 2 into one?
```suggestion
        $this->sortable(true, 'timecreated', SORT_DESC);
```